python/cudf_kafka/cudf_kafka/_lib/kafka.pxd

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from libcpp.string cimport string
+from libcpp.map cimport map
+from libc.stdint cimport int64_t
+from cudf._lib.io.datasource cimport Datasource
+
+
+cdef extern from "kafka_consumer.hpp" \
+        namespace "cudf::io::external::kafka" nogil:
+
+    cpdef cppclass kafka_consumer:
+
+        kafka_consumer(map[string, string] configs,
+                       string topic_name,
+                       int partition,
```

Comment:
Can we use an `int32_t` or something similar instead of `int`. I believe `int` is platform specific which can get us in trouble.

Reply:
sure thing